### ISSUE TITLE
Option to specify default/home channel which is shown instead of home page `config.json` …

### ISSUE THREAD

**Issue by fooness** [vector-im/element-web#15007]
From https://github.com/vector-im/element-web/blob/develop/docs/config.md …

> 17. embeddedPages: Configures the pages displayed in portions of Element that embed static files, such as:
    […]
    homeUrl: Content shown on the inside of the app when a specific room is not selected. By default, no home page is configured. If one is set, a button to access it will be shown in the top left menu.

It would be helpful if we could define a default/home channel, too. Instead of showing the home page after logging in or after removing the `room/…` part from the URL, the specified channel would be shown. That channel might be read only and filled with content/information. Admins/Mods without access to the server could change the content here.

**Comment by t3chguy** [comment]
There are many edge cases here.

What if the room is not joined?
What if the room is not accessible?

**Comment by fooness** [comment]
Seems I forgot to mention that this channel would be `auto_join`ed … but you are right, what if someone decides do leave the channel. Why not, in such a case, have it fall back to the default home page.

**Comment by fooness** [comment]
Another approach would be to auto-show that channel **only once after register** … somewhat similar to how `welcomeUserId` seemed to work, but with a channel instead of a bot/user. (“seemed to work” because, if I remember correctly, someone in Matrix HQ pointed out that this feature is somewhat deprecated.)

**Comment by t3chguy** [comment]
Well the auto joined channels are governed by the server (Synapse)
Element can be used with any server, including those which have no auto-join configured.
Synapse also doesn't notify Element which rooms are configured as auto-join.

**Comment by fooness** [comment]
You’re right, I also I just realize that this would rather be an option for synapse, so it’s client independent … but for this to work, probably the client somehow needs to communicate with synapse to figure out if such a setting exists …

“auto_join this and that and that other channel and, after the very first login, greet the user with this channel”
